### PR TITLE
Fixed admin javascript reload functions

### PR DESCRIPTION
**Pull request by gpickin**
https://ortussolutions.atlassian.net/browse/CONTENTBOX-842

### CODE REVIEW

**Comment by gpickin** [comment]
FYI - injecting the interceptor didn't work. Thats why i used the application scope. 
If you have a fix @lmajano - I would like to know.

File: modules/contentbox/models/ui/AdminMenuService.cfc [hunk 1]
```diff
@@ -175,7 +175,7 @@ component accessors="true" threadSafe singleton{
 		var xehDoLogout			= "#this.ADMIN_ENTRYPOINT#.security.doLogout";
 		var xehAutoUpdates		= "#this.ADMIN_ENTRYPOINT#.autoupdates";
 		var xehAbout			= "#this.ADMIN_ENTRYPOINT#.dashboard.about";
-		var xehAdminAction		= "/#this.ADMIN_ENTRYPOINT#/dashboard/reload";
+        var xehAdminAction        = "#application.wirebox.getInstance( 'interceptor-ses' ).getBaseURL()#/#this.ADMIN_ENTRYPOINT#/dashboard/reload";
```
Comment (lmajano) [inline modules/contentbox/models/ui/AdminMenuService.cfc]: Just change this to `event.getSESBaseURL()`

Comment (gpickin) [inline modules/contentbox/models/ui/AdminMenuService.cfc]: I didn't see event there :) 
Ok, testing and will push.

Comment (gpickin) [inline modules/contentbox/models/ui/AdminMenuService.cfc]: Ok, fixed. 
I wonder how many other people had issues with no rewrites.

**Comment by gpickin** [comment]
Gunnar just made a pull request with a build link instead. Which solution do you prefer?

**Comment by lmajano** [comment]
Ok, will go with the other one thanks!!

**Pull request closed without merging**